src/app/components/main-list/main-list.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="main-list-container center">
+    
+    <div>
+        <h2>ADS</h2>
+        <!-- <p><img src="assets/img/geographer-ad.png" alt=""></p> -->
+        <p>{{this.adName}}</p>
+        <p><img [src]="adUrl"></p>
+        <!-- <div style="height: 500px">
+            <h2>ADS</h2>
```

Comment:
lets get rid of all the comments that we don't need anymore.  Only keep comments that are explanatory or for future features with clear notes that they are for future features and what they need in order to be implemented.